### PR TITLE
Setup view framework

### PR DESCRIPTION
**Pull request by susiecoleman**
Produced a base view template which all other views will build on

### CODE REVIEW

**Comment by susiecoleman** [comment]
[Navbar.pdf](https://github.com/guardian/identity-admin/files/9412/Navbar.pdf)

File: app/assets/stylesheets/override.less [hunk 1]
```diff
@@ -0,0 +1,21 @@
+@brand-primary: #005689;
+
+//Navbar
+@navbar-default-bg: @brand-primary;
+@navbar-border-radius: 0px;
+
+.navbar-form {
```
Comment (emma-p) [inline app/assets/stylesheets/override.less]: Maybe we can keep all the custom css into `custom.css` and keep in that file only the variables we override from bootstrap

Comment (susiecoleman) [inline app/assets/stylesheets/override.less]: Moved

**Comment by emma-p** [comment]
:+1:

**Pull request merged**